jboss-eap7-with-helm.yaml

```diff
@@ -62,7 +62,8 @@ spec:
 
         1.  Click **Install** to create the JBoss EAP 7 application using the Helm Chart.
 
-            - The console switches to the [Topology]{{highlight qs-nav-topology}} view and display a note about the Helm release that has been installed.
+            - The console switches to the [Topology]{{highlight qs-nav-topology}} view and display a note about the Helm release that has been installed.  
+              Once you have read this note, you can close it.
```

Comment:
- The console opens the [Topology]{{highlight qs-nav-topology}} view and displays a note that has details about the Helm release.  
 After you read the note, you can close it.
 
 @jmesnil One question:

 **1. Is the following content the release note that has details about the Helm release? **
 
```
 Your EAP 7.4 application is building! To follow the build, run:

$ oc get build -w

Note that your Deployment will report "ErrImagePull" and "ImagePullBackOff" until the build is complete. Once the build is complete, your image will be automatically rolled out.

To follow the deployment of your application, run:

$ oc get deployment eap74 -w
 
```

Reply:
updated the instructions.

@sangeetaraghu the release notes come from inside the Helm Chart at https://github.com/jbossas/eap-charts/blob/main/charts/eap74/templates/NOTES.txt

Reply:
Thank you @jmesnil